locales/templates/app.pot

```diff
@@ -404,3 +414,28 @@ msgctxt "exit_firefox_a11y"
 msgid "Exit %1$s"
 msgstr ""
 
+#. Hint Strings
+#. Hint text displayed when the overlay is open.      %IMAGE will be replaced
+#. with the image of a hardware back button.      When this button is pressed,
+#. the overlay will close and the user will be returned to the browser.
+#, c-format
+msgctxt "hint_press_back_to_close_overlay"
+msgid "Press %IMAGE to return"
+msgstr ""
+
+msgctxt "hint_press_back_to_close_overlay_a11y"
+msgid "Press the back button to return"
+msgstr ""
+
+#. Hint text displayed when the browser is open.      %IMAGE will be replaced
+#. with the image of a hardware menu button.      When this button is pressed,
+#. the overlay will open and the browser will be hidden underneath it
+#, c-format
+msgctxt "hint_press_menu_to_open_overlay"
+msgid "Press %IMAGE for Firefox TV Home, search, settings, and more"
```

Comment:
We usually go with the placeholder '%1$s' instead of calling out the app name.
I think I found a couple instances in the strings though where that's not the case, it's probably worth correcting in an upcoming export:
firefox_tv_brand_name
notification_pinned_site
notification_unpinned_site